creating-virtual-machines/interfaces-and-networks.adoc

```diff
@@ -474,10 +528,10 @@ spec:
       networkName: sriov-net-crd
 ----
 
-__________________________________________________________________________________
-*Note:* you need to enable the SRIOV feature gate to use the feature.
-For example:
-__________________________________________________________________________________
+______________________________________________________________________________
+*Note:* you need to enable the SRIOV feature gate to use the feature.  For
```

Comment:
Two spaces :(

Reply:
...which is bad because?..

Reply:
Removed two spaces. Batman can rest now!